setup.py

```diff
@@ -0,0 +1,41 @@
+from setuptools import setup
+
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "Natural Language :: English",
+    "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
```

Comment:
The license checked into the repo is MIT. Other Mint repos are BSD-3 so would prefer to use that for consistency, unless there's a good reason not to.

Reply:
No reason. I'll switch it.